README.md

```diff
@@ -56,18 +65,38 @@ R_eff is estimated by running a Kalmar filter estimator with a nonlinear SIR-bas
 For raised warnings and errors, check the readme section first, for basic troubleshooting.
 If the problem persist, please contact the developers.
 
-
-
 > Copyright 2020 Luxembourg Centre for Systems Biomedicine
 >
 > Licensed under the Apache License, Version 2.0 (the "License");
->  you may not use this file except in compliance with the License.
->  You may obtain a copy of the License at
+> you may not use this file except in compliance with the License.
+> You may obtain a copy of the License at
 >
->    http://www.apache.org/licenses/LICENSE-2.0
+> http://www.apache.org/licenses/LICENSE-2.0
 >
->  Unless required by applicable law or agreed to in writing, software
->  distributed under the License is distributed on an "AS IS" BASIS,
->  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
->  See the License for the specific language governing permissions and
->  limitations under the License.
+> Unless required by applicable law or agreed to in writing, software
+> distributed under the License is distributed on an "AS IS" BASIS,
+> WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+> See the License for the specific language governing permissions and
+> limitations under the License.
+
+## How to run the pipeline
+
+Build the docker image using:
+
+```bash
+docker build -t covid19 .
+```
+
+Then, run the analysis:
+
+```bash
+docker run -v $(pwd)/output:/covid19-reproductionNumber/output covid19-reproductionNumber
+```
+
+Run the pipeline properly speaking
+
+From the root directory, run:
+
+```bash
+./run_pipeline.sh
+```
```

Comment:
We could simplify the invocation of some tasks by providing `Makefile` with:
`make build`
`make clean`

I'm not a biggest fan of these (I prefer plain readme.md :)), but it might come handy in case we'd have more of these snippets

Reply:
nice idea - just pushed it 😄